core/src/main/java/io/lionweb/lioncore/java/self/LionCore.java

```diff
@@ -1,118 +1,196 @@
     return getInstance().requireConceptByName("Reference");
   }
 
-  public static Language getInstance() {
-    if (INSTANCE == null) {
-      INSTANCE = new Language("LionCore_M3");
-      INSTANCE.setID("-id-LionCore-M3");
-      INSTANCE.setKey("LionCore-M3");
-      INSTANCE.setVersion(JsonSerialization.DEFAULT_SERIALIZATION_FORMAT);
+  public static @Nonnull Concept getAnnotation(@Nonnull LionWebVersion lionWebVersion) {
+    return getInstance(lionWebVersion).requireConceptByName("Annotation");
+  }
+
+  public static @Nonnull Concept getConcept(@Nonnull LionWebVersion lionWebVersion) {
+    return getInstance(lionWebVersion).requireConceptByName("Concept");
+  }
+
+  public static @Nonnull Concept getInterface(@Nonnull LionWebVersion lionWebVersion) {
+    return getInstance(lionWebVersion).requireConceptByName("Interface");
+  }
+
+  public static @Nonnull Concept getContainment(@Nonnull LionWebVersion lionWebVersion) {
+    return getInstance(lionWebVersion).requireConceptByName("Containment");
+  }
+
+  public static @Nonnull Concept getDataType(@Nonnull LionWebVersion lionWebVersion) {
+    return getInstance(lionWebVersion).requireConceptByName("DataType");
+  }
+
+  public static @Nonnull Concept getEnumeration(@Nonnull LionWebVersion lionWebVersion) {
+    return getInstance(lionWebVersion).requireConceptByName("Enumeration");
+  }
+
+  public static @Nonnull Concept getEnumerationLiteral(@Nonnull LionWebVersion lionWebVersion) {
+    return getInstance(lionWebVersion).requireConceptByName("EnumerationLiteral");
+  }
+
+  public static @Nonnull Concept getFeature(@Nonnull LionWebVersion lionWebVersion) {
+    return getInstance(lionWebVersion).requireConceptByName("Feature");
+  }
+
+  public static @Nonnull Concept getClassifier(@Nonnull LionWebVersion lionWebVersion) {
+    return getInstance(lionWebVersion).requireConceptByName("Classifier");
+  }
+
+  public static @Nonnull Concept getLink(@Nonnull LionWebVersion lionWebVersion) {
+    return getInstance(lionWebVersion).requireConceptByName("Link");
+  }
+
+  public static @Nonnull Concept getLanguage(@Nonnull LionWebVersion lionWebVersion) {
+    return getInstance(lionWebVersion).requireConceptByName("Language");
+  }
+
+  public static @Nonnull Concept getLanguageEntity(@Nonnull LionWebVersion lionWebVersion) {
+    return getInstance(lionWebVersion).requireConceptByName("LanguageEntity");
+  }
+
+  public static @Nonnull Concept getPrimitiveType(@Nonnull LionWebVersion lionWebVersion) {
+    return getInstance(lionWebVersion).requireConceptByName("PrimitiveType");
+  }
+
+  public static @Nonnull Concept getProperty(@Nonnull LionWebVersion lionWebVersion) {
+    return getInstance(lionWebVersion).requireConceptByName("Property");
+  }
+
+  public static @Nonnull Concept getReference(@Nonnull LionWebVersion lionWebVersion) {
+    return getInstance(lionWebVersion).requireConceptByName("Reference");
+  }
+
+  public static @Nonnull Language getInstance() {
```

Comment:
What about declaring this method `@Deprecated`? It's almost certainly wrong to use it.

Same for all the other version-less methods.